src/google-maps/google-map/google-map.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+  OnInit,
+} from '@angular/core';
+import {ReplaySubject} from 'rxjs';
+
+/**
+ * Angular component that renders a Google Map via the Google Maps JavaScript
+ * API.
+ * @see https://developers.google.com/maps/documentation/javascript/reference/
+ */
+@Component({
+  selector: 'google-map',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<div class="map-container"></div>',
+})
+export class GoogleMap implements OnInit {
+  // Arbitrarily chosen default size
+  @Input() height = '500px';
+  @Input() width = '500px';
+
+  // TODO(mbehrlich): add options, handlers, properties, and methods.
```

Comment:
I assume that updating the map's height/width in `ngOnChanges()` is the kind of thing included with this TODO?

Reply:
Yes, although my plan was to avoid ngOnChanges if I could. See my demo for an example of what I was planning: https://github.com/mbehrlich/map-demo/blob/master/src/map/map.component.ts#L62.

I think this is a more readable pattern personally.

Reply:
OK. Were you planning to add height/width to `MapOptions` or hook them up to separate getters/subjects?

Reply:
They will remain separate, the reason being that the height and width are not included in MapOptions in the Google Maps JavaScript API.